setup.py

```diff
@@ -8,20 +8,29 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+# Make sources available using relative paths from this file's directory.
+os.chdir(os.path.dirname(os.path.abspath(__file__)))
+
+
+def load_version():
     """Returns the version found in nilearn/version.py
 
     Importing nilearn is not an option because there may dependencies
     like nibabel which are not installed and setup.py is supposed to
     install them.
     """
-    locals_dict = {}
+    # load all vars into globals, otherwise
+    #   the later function call using global vars doesn't work.
     globals_dict = {}
-    with open(os.path.join('nilearn', 'version.py')) as fp:
-        exec(fp.read(), globals_dict, locals_dict)
+    execfile(os.path.join('nilearn', 'version.py'), globals_dict)
```

Comment:
Took me a while to remember why we used exec rather than execfile and the reason is execfile doesn't exist in Python 3.